ui/components/app/signature-request/signature-request.container.js

```diff
@@ -157,8 +165,19 @@ mapDispatchToProps = function (dispatch) {
         }),
       );
     },
-    cancelAll: (unconfirmedMessagesList) =>
-      dispatch(cancelMsgs(unconfirmedMessagesList)),
+    cancelAllApprovals: (unconfirmedMessagesList) => {
```

Comment:
We can reuse the action I mentioned we could create in the other comment.

Reply:
Fixed